src/store/activities.ts

```diff
@@ -196,6 +194,23 @@ async function getRemoteTransaction(
   })
 }
 
+function decodeLogDataHandleOps(
+  transaction: ethers.providers.TransactionResponse
+): ethers.utils.Result {
+  const abi = [
+    'function handleOps((address,uint256,bytes,bytes,uint256,uint256,uint256,uint256,uint256,bytes,bytes)[],address)',
+  ]
+  const iface = new ethers.utils.Interface(abi)
+  return iface.decodeFunctionData('handleOps', transaction.data)
+}
+
+function getAmountUsingCallData(data: string): BigNumber {
+  const abi = ['function executeCall(address,uint256,bytes)']
+  const iface = new ethers.utils.Interface(abi)
+  const decodedData = iface.decodeFunctionData('executeCall', data)
+  return decodedData[1]
+}
+
```

Comment:
I guess we should keep these contract specific stuff in it's own utils so it can be made reusable whenever required.

Reply:
I think let's leave this for now here, if we need it somewhere then we can move it to utils @shrinathprabhu